elmslie-core/src/main/java/vivid/money/elmslie/core/store/Result.kt

```diff
@@ -13,8 +13,8 @@ data class Result<State : Any, Effect : Any, Command : Any>(
         command: Command? = null,
     ) : this(
         state = state,
-        effects = effect?.let(::listOf) ?: emptyList(),
```

Comment:
why this is required for KMP? yeah, this line is simplified but it also creates additional list for filtering

Reply:
under the hood it works exactly the same but looks more readble 
`public fun <T : Any> listOfNotNull(element: T?): List<T> = if (element != null) listOf(element) else emptyList()`